lib/controller.js

```diff
@@ -22,21 +26,22 @@ Tessel.list = function(opts) {
     // When a Tessel is found
     seeker.on('tessel', function displayResults(tessel) {
       var note = '';
-
       // Add it to our array
       foundTessels.push(tessel);
-
       // Add a note if the user isn't authorized to use it yet
       if (tessel.connection.connectionType === 'LAN' && !tessel.connection.authorized) {
         note = '(USB connect and run `tessel provision` to authorize)';
       }
-
       // Print out details...
       logs.basic(sprintf('\t%s\t%s\t%s', tessel.name, tessel.connection.connectionType, note));
     });
 
     // Called after CTRL+C or timeout
     function stopSearch() {
+      if (foundTessels.length > 1) {
+        debug('heuristics ? => found: ' + foundTessels.length + ' === 1 || (' + foundTessels.length + ' === 2 && ' + foundTessels[0].name + ' === ' + foundTessels[1].name + ' && ' + foundTessels[0].connections[0].connectionType + ' !== ' + foundTessels[1].connections[0].connectionType + ')');
```

Comment:
@Student007 I think we should remove it and add infrastructure to `controller.list` to weed out duplicates.